docs/reference/connectors.md

```diff
@@ -0,0 +1,91 @@
+# Connectors
+
+This resource packs supports the following connection configurations:
+
+* Environment Variables
+* Credentials File
+* Managed Service Identity Connector
+
+For any of these connection strategies you'll need a Subscription ID. Your Subscription ID may be found by ... When you run InSpec it will be included in the target. `-t azure://$SUBSCRIPTION_ID`.
```

Comment:
This is incomplete.  Also, don't you need all four pieces of data (subscription id, client_id, tenant_id, and client_secret) to set this up?

Reply:
Thank you! 

At the moment yes you need all four pieces of information (and in general). We have another PR in flight #77 that will use Train for the backend and enable these additional options. You could use the MSI connector in Azure, which would only require a Tenant ID and Subscription ID.